### ISSUE TITLE
Add the viridis colormap to the available LUTs

### ISSUE THREAD

**Issue by daytonb** [enthought/mayavi#407]
I'd like to use the viridis colormap (matplotlib 2.0's default colormap) in mayavi. I'd like to see it as one of the shipping LUTs.

If that's not an option, how would I go about using the viridis colormap to create a custom LUT?

**Comment by prabhuramachandran** [comment]
This would be a great PR to send and is really easy to do:

1. Install the latest matplotlib.
2. In the mayavi source tree, `cd scripts`
3. Run the cm2lut.py script
4. Run the gen_lut_previews.py script

This will update the luts and also generate a bunch of new .gif files in the source tree.  Add these files and send a PR and this can be merged.  Thanks!

**Comment by daytonb** [comment]
I put in a pull request, but it needs someone else to run a script to generate the gifs or some follow on help to deal with a savefig issue. https://github.com/enthought/mayavi/pull/410.

**Comment by daytonb** [comment]
I was able to get the gifs updated. On my local repo I merged in pull request https://github.com/enthought/mayavi/pull/350, which allowed me to get savefig working. Then I cherry-picked the gifs over to my viridis-lut branch.

**tldr:** pull request https://github.com/enthought/mayavi/pull/410 should be ready to merge adding all of the new matplotlib colorschemes to mayavi LUTs.

**Comment by prabhuramachandran** [comment]
Thanks for the PR, which is now merged.
